client/my-sites/plans/jetpack-plans/product-store/items-list/most-popular.tsx

```diff
@@ -33,6 +36,11 @@ export const MostPopular: React.FC< MostPopularProps > = ( {
 		getOnClickPurchase,
 		isMultisite,
 	} = useStoreItemInfoContext();
+	const translate = useTranslate();
+
+	const isEnglish = config< Array< string | undefined > >( 'english_locales' ).includes(
```

Comment:
I believe you can add the label `String Freeze` to your PR to ensure new strings are translated before merging.

Reply:
Added. Thanks